installation/index.qmd

```diff
@@ -45,7 +45,7 @@ To get the most out of Viash, you'll also need to install [Docker](https://docs.
 To install Viash to a `bin` folder in your current directory, run the following command to download and run the install script:
 
 ```bash
-wget -qO- dl.viash.io | bash; mv viash $HOME/bin
+wget -qO- dl.viash.io | bash; mv viash $HOME/.local/bin
```

Comment:
How about we install it directly into `/usr/local/bin`? (Example: https://www.nextflow.io/docs/latest/install.html#install-nextflow).

Since Viash will automatically switch to the version of Viash specified in the project config, updating the Viash executable is no longer really necessary.